MediaBrowser.MediaEncoding/Subtitles/SubtitleEncoder.cs

```diff
@@ -511,12 +511,21 @@ private async Task ConvertTextSubtitleToSrtInternal(string inputPath, string lan
 
             await semaphore.WaitAsync(cancellationToken).ConfigureAwait(false);
 
+            var subtitleStreamIndex = mediaSource.MediaStreams.Where(i => i.Path == subtitleStream.Path).ToList().IndexOf(subtitleStream);
```

Comment:
Can't we place `FindIndex` somewhere that is visible for both projects?

Reply:
Somewhere in `Jellyfin.Extensions` maybe or do we have a internal untility namespace?

Reply:
It's very specific to MediaStreams and Jellyfin.Extensions is for more generic stuff. Just make it public in EncodingHelper and use it.